modules/abstract-cosmos/src/cosmosCoin.ts

```diff
@@ -129,7 +129,7 @@ export class CosmosCoin extends BaseCoin {
         throw new Error('Tx outputs does not match with expected txParams recipients');
       }
       // WithdrawDelegatorRewards transaction doesn't have amount
-      if (transaction.type !== TransactionType.StakingWithdraw) {
+      if (transaction.type !== TransactionType.StakingWithdraw && transaction.type !== TransactionType.ContractCall) {
```

Comment:
Contract interaction support the amount, anyway that will not require in our case but let's keep in mind.

Reply:
yeah, it is under funds and didn't want to update current amount validation for it